.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+        - repo: https://github.com/ambv/black
```

Comment:
replace these spaces for [2 paces](https://docs.saltstack.com/en/master/topics/troubleshooting/yaml_idiosyncrasies.html#:~:text=The%20suggested%20syntax%20for%20YAML,uniform%20and%20not%20deeply%20nested.):

*The suggested syntax for YAML files is to use 2 spaces for indentation, but YAML will follow whatever indentation system that the individual file uses. Indentation of two spaces works very well for SLS files given the fact that the data is uniform and not deeply nested.*